src/mscorlib/model.xml

```diff
@@ -8476,6 +8476,7 @@
       <Member Name="GetTypeFromCLSID(System.Guid,System.String)" />
       <Member Name="GetTypeFromCLSID(System.Guid,System.String,System.Boolean)" />
       <Member Name="GetTypeFromHandle(System.RuntimeTypeHandle)" />
+      <Member Name="GetTypeFromHandleUnsafe(System.IntPtr)" />
```

Comment:
Could you please mark it with `Status="ImplRoot"` and add comment `<!-- Used by System.Linq.Expressions tests -->` next to.